README.md

```diff
@@ -1,2 +1,77 @@
-# connectiq-apps
-A collection of Connect IQ apps and libraries.
+# /connectiq-apps
+_A collection of Connect IQ apps and libraries_
+
+------------------
+
+## Purpose
+This repo is officially maintained by the Garmin&reg; Connect IQ team. We are happy to have you here. It is our hope that this repository will extend the  resources available to our developers to help them build exciting new content!
```

Comment:
Double space in "extend the__resources available"

Reply:
Done